packages/lesswrong/components/bookmarks/ReadHistoryTab.tsx

```diff
@@ -1,9 +1,9 @@
-import React, {useState} from 'react';
-import { registerComponent, Components, fragmentTextForQuery } from '../../lib/vulcan-lib';
-import {AnalyticsContext} from "../../lib/analyticsEvents";
-import {useCurrentUser} from "../common/withUser"
-import { gql, useQuery, NetworkStatus } from '@apollo/client';
-import moment from 'moment';
+import React, {useState} from 'react'
+import {Components, fragmentTextForQuery, registerComponent} from '../../lib/vulcan-lib'
+import {AnalyticsContext} from '../../lib/analyticsEvents'
+import {useCurrentUser} from '../common/withUser'
+import {gql, NetworkStatus, useQuery} from '@apollo/client'
+import moment from 'moment'
```

Comment:
I'm confused by all the diffs here (since none of them seem like they change anything except formatting)

Reply:
uh, it's my IDE changing the formatting of all imports when I do auto-import, will look into disabling it =\